docs/user_guide/installation_and_version_guide/v0.7.1.rst

```diff
@@ -8,7 +8,7 @@ v0.7.1
 
 * Relaxed checks made on :code:`X` in :code:`_validate_and_reformat_input()`
   since that is the concern of the underlying estimator and not Fairlearn.
-* Add support for Python 3.9
+* Add support for Python 3.9 and 3.10, deprecate support for Python 3.6
```

Comment:
should go to version 0.8, and "remove" instead of "deprecate".

Reply:
Will do!